templates/init/dataconnect/schema.gql

```diff
@@ -1,28 +1,46 @@
-# # Example schema for simple email app
-# type User @table(key: "uid") {
-# 	uid: String!
-# 	name: String!
-# 	address: String!
+# # Example schema for simple movie review app
+
+# # Users
+# # Suppose a user can leave reviews for movies
+# # user  -> reviews is a one to many relationship,
+# # movie -> reviews is a one to many relationship
+# # movie <-> user is a many to many relationship
+# type User @table {
+#   id: String! @col(name: "user_auth")
+#   username: String! @col(name: "username", dataType: "varchar(50)")
+#   # The following are generated by the user: User! field in the Review table
+#   # reviews_on_user 
+#   # movies_via_Review
 # }
 
-# type Email @table {
-# 	subject: String!
-# 	sent: Date!
-# 	text: String!
-# 	from: User!
+# # Movies
+# type Movie @table {
+#   # The below parameter values are generated by default with @table, and can be edited manually.
+#   # implicitly calls @col to generates a column name. ex: @col(name: "movie_id")
```

Comment:
```suggestion
#   # implies directive `@col(name: "movie_id")`, generating a column name
```

I'd avoid the word "call" with directives since directives aren't functions or methods.